README.md

```diff
@@ -262,7 +262,7 @@ module Translation
   # Australian, New Zealand variations is provided.
   class PrimAndProper
     def initialize
-      @converters = { :en => { :"en-AU" => AmericanToAustralian.new,
+      @converters = { en: { :"en-AU" => AmericanToAustralian.new,
```

Comment:
If this PR goes through, please also update the affected indentation (the next 3 lines). I'll only comment here, with the assumption you'll look for other cases where the indentation got messed up.